packages/select/src/Option/Options.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { css } from '@leafygreen-ui/emotion';
+import { descriptionClassName } from '@leafygreen-ui/input-option';
+import { createUniqueClassName } from '@leafygreen-ui/lib';
+import { fontFamilies, transitionDuration } from '@leafygreen-ui/tokens';
+
+export const OptionClassName = createUniqueClassName('option');
+
+export const optionStyle = css`
+  display: flex;
+  width: 100%;
+  outline: none;
+  overflow-wrap: anywhere;
+  transition: background-color ${transitionDuration.default}ms ease-in-out;
+  position: relative;
+  padding: 8px 12px;
+  line-height: inherit;
+
+  &:before {
+    content: '';
+    position: absolute;
+    transform: scaleY(0.3);
+    top: 7px;
+    bottom: 7px;
```

Comment:
This feels kinda bad. I should assume `8px` would work just as well here

Reply:
i didn't add this so im going to leave as is. We can address this during the consistency work.